vivisect/qt/funcgraph.py

```diff
@@ -277,6 +301,31 @@ def _hotkey_deczoom(self):
         #self.vw.vprint("NEW ZOOM    %f" % newzoom)
         self.mem_canvas.setZoomFactor(newzoom)
 
+    def refresh(self):
+        '''
+        Cause the Function Graph to redraw itself.
+        This is particularly helpful because comments and name changes don't
+        immediately display.  Perhaps someday this will update only the blocks
+        that have changed since last update, and be fast, so we can update
+        after every change.  
+        '''
+        self._last_viewpt = self.mem_canvas.page().mainFrame().scrollPosition()
+        self.clearText()
+        self.fva = None
+        self._renderMemory()
+
+    @workthread
```

Comment:
having to call the workthread does seem odd, but it is required.  i'd love for you to tell me why :)  there seems to be a sequencing issue where switching threads causes the effects to occur in the main qt thread after the drawing is done, but without the thread context switch, the update seems to be called before the view is rendered sufficiently, and effectively drops the viewport positioning update.